src/app/shared/pipes/newline-split.pipe.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This pipe takes a string and splits it into an array of strings based on newlines.
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * This pipe splits a string by newline characters or escaped sequences.
+ */
+@Pipe({
+  name: 'newlineSplit',
+})
+export class NewlineSplitPipe implements PipeTransform {
```

Comment:
I would put the verb first, i.e. SplitNewline. In Python it's called splitlines so why not SplitLinesPipe?